### PR TITLE
AC: added shuffle option for creation subset

### PR DESCRIPTION
**Pull request by Anna-Mironova**

### CODE REVIEW

File: tools/accuracy_checker/README.md [hunk 1]
```diff
@@ -97,6 +97,7 @@ You may refer to `-h, --help` to full list of command line options. Some optiona
 - `--num_requests` number requests for async execution. Allows override provided in config info. Default is `AUTO`
 - `--model_attributes` directory with additional models attributes.
 - `--subsample_size` dataset subsample size.
+- `--shuffle` allow shuffle annotation during creation a subset.
```
Comment (eaidova) [inline tools/accuracy_checker/README.md:100]: please say  2-3 wordsabout default value and that it works only if subsample_size enabled

File: tools/accuracy_checker/tests/test_dataset.py [hunk 2]
```diff
@@ -288,7 +288,7 @@ def test_annotation_conversion_closer_to_zero_subset_ratio(self, mocker):
             'accuracy_checker.dataset.make_subset'
         )
         Dataset(config)
-        subset_maker_mock.assert_called_once_with(converted_annotation, 1, 666)
+        subset_maker_mock.assert_called_once_with(converted_annotation, 1, 666, True)
```
Comment (eaidova) [inline tools/accuracy_checker/tests/test_dataset.py:291]: please add tests with shuffle=False

File: tools/accuracy_checker/accuracy_checker/main.py [hunk 3]
```diff
@@ -245,6 +245,11 @@ def build_arguments_parser():
         required=False,
         type=str
     )
+    parser.add_argument(
+        '--shuffle', help="Allow shuffle annotation during creation a subset",
+        required=False,
+        type=cast_to_bool
+    )
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/main.py:252]: this not only way to get annotation, please also update convert.py in annotation_converters

File: tools/accuracy_checker/README.md (hunk 1 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/README.md]: Done

File: tools/accuracy_checker/accuracy_checker/main.py (hunk 3 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/accuracy_checker/main.py:252]: New argument is added in convert.py

**Pull request merged**